src/oc/web/utils/section.cljs

```diff
@@ -0,0 +1,14 @@
+(ns oc.web.utils.section
```

Comment:
Having these functions in `utils` instead of `api` is a bit confusing. Please move to `api`